Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/NIDCPowerSourceDCVoltage.lvlib

```diff
@@ -0,0 +1,34 @@
+
+</Property>
+	<Item Name="Advanced" Type="Folder">
+		<Item Name="Build Assets" Type="Folder">
+			<Item Name="Post-Build Action.vi" Type="VI" URL="../Advanced/Build Assets/Post-Build Action.vi"/>
+		</Item>
+		<Item Name="MeasurementPlugin.lvclass" Type="LVClass" URL="../Advanced/MeasurementPlugin.lvclass"/>
+	</Item>
+	<Item Name="subVIs" Type="Folder">
+		<Item Name="Convert Location to Address String.vi" Type="VI" URL="../subVIs/Convert Location to Address String.vi"/>
+		<Item Name="DCPower Specific Logic.vi" Type="VI" URL="../subVIs/DCPower Specific Logic.vi"/>
+		<Item Name="Location to Grpc Device.vi" Type="VI" URL="../subVIs/Location to Grpc Device.vi"/>
+	</Item>
+	<Item Name="TestStand Code Modules" Type="Folder">
+		<Item Name="Create and Register Sessions.vi" Type="VI" URL="../TestStand Code Modules/Create and Register Sessions.vi"/>
+		<Item Name="Unregister and Destroy Sessions.vi" Type="VI" URL="../TestStand Code Modules/Unregister and Destroy Sessions.vi"/>
+		<Item Name="Update Pin Map.vi" Type="VI" URL="../TestStand Code Modules/Update Pin Map.vi"/>
+	</Item>
+	<Item Name="Get Measurement Details.vi" Type="VI" URL="../Get Measurement Details.vi"/>
+	<Item Name="Get Type Specializations.vi" Type="VI" URL="../Get Type Specializations.vi"/>
+	<Item Name="Get UI Details.vi" Type="VI" URL="../Get UI Details.vi"/>
+	<Item Name="Measurement Configuration.ctl" Type="VI" URL="../Measurement Configuration.ctl"/>
+	<Item Name="Measurement Logic.vi" Type="VI" URL="../Measurement Logic.vi"/>
```

Comment:
FYI: Consider doing the ResolveService calls in parallel, at least for Measurement Logic.vi.

Reply:
I parallelized it in the measurement logic.